shuttle/driver_logs/__init__.py

```diff
@@ -0,0 +1,34 @@
+from flask import render_template
+from flask_classy import FlaskView, route, request
+from datetime import datetime
```

Comment:
Make sure to split up imports into three different sections.

The first section should be global packages (packages that come with python like datetime, json, etc)

The second section should be packages from our virtual environment (things in requirements.txt)

The third section should be stuff from our project (anything from shuttle...)

These three sections should have a space between each section.